css-three-column-layout/styles.css

```diff
@@ -0,0 +1,70 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-basis: auto;
+}
+
+h2 {
+  font-family: Lato, sans-serif;
+  font-weight: 400;
+  color: white;
+  text-align: center;
+  padding-top: 5vh;
+  padding-bottom: 30px;
+}
+
+body {
+  background-color: #009ffd;
+}
+
+h3 {
+  font-family: Lato, sans-serif;
+  font-weight: 400;
+  color: white;
+}
+
+p {
+  font-family: Lato, sans-serif;
+  font-weight: 400;
+  color: lightgray;
+  font-size: 16px;
+}
+
+.column-1 {
+  width: 33.33%;
+  text-align: center;
+  float: left;
+}
+
+.column-2 {
+  width: 33.33%;
+  float: left;
+  text-align: center;
+}
+
+.column-3 {
+  width: 33.33%;
+  float: left;
+  text-align: center;
+}
```

Comment:
All three of these classes do the same thing. You should just make one class and reuse it, named probably `column-one-third`.

Also, the only styling that should be on a `column` class is `width` or `flex-basis`. Make sure the measurement accurately matches the name of the class as well 🤓 . If you need any other additional styling on top of `width` or `flex-basis`, you should make a separate class that you can add to the element in addition to the `column` class.